.github/workflows/container-build.yml

```diff
@@ -0,0 +1,17 @@
+name: Container
+
+on: [ push, pull_request ]
+
+jobs:
+  publisher:
+    name: Build
+    runs-on: ubuntu-latest
```

Comment:
I wonder if ubuntu is the best choice here.  Seems like a lot of baggage.

Reply:
I just restore the file that I deleted before